libs/ledgerjs/packages/cryptoassets/src/crypto-assets-importer/importers/apt/index.ts

```diff
@@ -0,0 +1,63 @@
+import fs from "fs";
+import path from "path";
+import { fetchTokensFromCALService } from "../../fetch";
+
+type AptosToken = [
+  string, // id
+  string, // ticker
+  string, // name
+  string, // contractAddress
+  number, // decimals
+  boolean?, // delisted
+];
+
+export const importAptosTokens = async (outputDir: string, standard: string) => {
+  try {
+    console.log(`importing aptos ${standard} tokens...`);
+    const { tokens, hash } = await fetchTokensFromCALService(
+      { blockchain_name: "aptos", standard: standard },
+      ["id", "ticker", "name", "contract_address", "decimals", "delisted"],
+    );
+    const aptosTokens: AptosToken[] = tokens.map(token => {
+      return [
+        token.id,
+        token.ticker,
+        token.name,
+        token.contract_address,
+        token.decimals,
+        token.delisted,
+      ];
+    });
```

Comment:
```suggestion
    const aptosTokens: AptosToken[] = tokens.map(token => ([
        token.id,
        token.ticker,
        token.name,
        token.contract_address,
        token.decimals,
        token.delisted,
      ]));
```

this is just an aesthetics suggestion